### DISCUSSION TITLE
Cerium Delta v1.1.0

### DISCUSSION THREAD

**Discussion by NEURAL-Y** [NEURAL-Y/Cerium-Delta#16] [Announcements]
# Cerium Delta v1.1.0

Cerium Delta v1.1.0 introduces a dedicated **statistical analysis and static visualization layer** for Neural Vitality analysis.

This release extends Cerium Delta beyond the core Neural Vitality metrics by providing tools to statistically inspect and visually analyze layer-level **Layer Contribution Score (LCS), Sensitivity, and Evolution** data.

### Static Visualization

Added the `visualizer` interface for generating static analytical plots using Matplotlib and Seaborn.

Supported visualization workflows include:

* Bar plots
* IQR-based bar analysis
* Box plots
* Histograms
* Linear fitting plots
* Median Absolute Deviation (MAD) plots
* Correlation and covariance scatter plots
* Large-distribution visualization with Datashader

Visualization methods can operate on different Neural Vitality families, including weights and biases across LCS, Sensitivity, and Evolution measurements.

### Statistical Analysis

Added the `Statistical` interface for statistical processing used by the visualization layer.

The release includes analysis workflows involving:

* Pearson correlation
* Covariance
* Skewness
* Kurtosis
* Interquartile Range (IQR)
* Median Absolute Deviation (MAD)
* Linear regression

### Data & Visualization Dependencies

The visualization and statistical layer now uses:

* NumPy
* SciPy
* Pandas
* Matplotlib
* Seaborn
* Datashader

### Expanded Public API

The package public API now exposes:

```python
from cerium_delta import NVS, bridge, visualizer, Statistical
```

`NVS` remains responsible for Neural Vitality analysis, while `bridge` provides framework integration and `visualizer` and `Statistical` provide analytical inspection capabilities.

### Optional Framework Support

Cerium Delta continues to provide optional integrations for:

* PyTorch
* TensorFlow
* JAX
* scikit-learn
* ONNX

## Installation

```bash
pip install cerium-delta
```

Framework-specific integrations can be installed independently through the available extras.

## Release Focus

Cerium Delta v1.1.0 focuses on turning Neural Vitality results into **inspectable statistical and visual information**, allowing users to examine layer-level behavior rather than relying only on the raw metric output.

Cerium Delta remains an **observability and analysis toolkit**, not a neural-network training or optimization framework.

**Full Changelog**: https://github.com/NEURAL-Y/Cerium-Delta/compare/v1.0.1...v1.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/NEURAL-Y/Cerium-Delta/releases/tag/v1.1.0'>Cerium Delta v1.1.0</a>.</em>
